kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/adapter/QuickConversationAdapter.java

```diff
@@ -250,6 +251,14 @@ public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
                     showConversationSourceIcon(channel, myholder.attachmentIcon);
                 }
 
+                if(TextUtils.isEmpty(myholder.messageTextView.getText()))
+                {
+                    myholder.messageTextView.setCompoundDrawablesWithIntrinsicBounds(R.drawable.ic_messageicon, 0, 0, 0);
+                    myholder.messageTextView.setCompoundDrawablePadding(20);
+                    myholder.messageTextView.setText(DEFAULT_MSG_BODY);
+                }
+                else myholder.messageTextView.setCompoundDrawablesWithIntrinsicBounds(0, 0, 0, 0);
```

Comment:
Have checked by creating more number of conversations,,so that you can scroll the recycler view & check this change is working fine or not

Reply:
Ya ya the else part is handling that thing . I have tested it with decent number of conversations .